tests/e2e/Services/Locale/LocaleBase.php

```diff
@@ -235,15 +235,15 @@ public function testLanguages(): array
             $response = $this->client->call(Client::METHOD_GET, '/locale/countries', [
                 'content-type' => 'application/json',
                 'x-appwrite-project' => $this->getProject()['$id'],
-                'x-appwrite-locale' => $lang,
+                'x-appwrite-locale' => $lang['code'],
             ]);
 
             if (!\is_array($response['body']['countries'])) {
                 throw new Exception('Failed to iterate locale: ' . $lang);
             }
 
             foreach ($response['body']['countries'] as $i => $code) {
-                $this->assertContains($code['code'], $defaultCountries, $code['code'] . ' country should be removed from ' . $lang);
+                $this->assertContains($code['code'], $defaultCountries, $code['code'] . ' country should be removed from ' . $lang['code']);
             }
 
             // foreach (array_keys($defaultCountries) as $i => $code) {
```

Comment:
Leftovers?

Reply:
Not sure, I think it was already there, I'll check

Reply:
Let's remove if not needed.